src/wasm2js.h

```diff
@@ -411,9 +411,8 @@ Ref Wasm2JSBuilder::processWasm(Module* wasm, Name funcName) {
     asmFunc[3]->push_back(processFunction(
```

Comment:
`params` and `vars` does not seem to be used anywhere, so we can delete them (not this line but the line above. Github doesn't let me comment on that line...)

Reply:
nice catch!